### PR TITLE
Bidtellect amp implementation…

### PR DESCRIPTION
**Pull request by edaquinta-bidtellect**

### CODE REVIEW

**Comment by edaquinta-bidtellect** [comment]
CLA have been signed 20 days ago. Company name is Bidtellect.

**Comment by jridgewell** [comment]
Looks like you'll need to rebase off current master.

**Comment by edaquinta-bidtellect** [comment]
??? This is brand new branch. I just branched from master and made the changes. Only one commit.

File: ads/bidtellect.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ */
+
+import {writeScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function bidtellect(global, data) {
+  const requiredParams = ['t','pid','sid'];
```
Comment (zhouyx) [inline ads/bidtellect.js]: nit: space after ,

File: ads/_config.js [hunk 2]
```diff
@@ -175,6 +175,10 @@ export const adConfig = {
     prefetch: 'https://s.ato.mx/p.js',
   },
 
+  bidtellect: {
+    renderStartImplemented: true,
```
Comment (zhouyx) [inline ads/_config.js]: I see you implement `render-start`, but when I test locally I am never able to receive it? Could you double check?

**Comment by edaquinta-bidtellect** [comment]
I made this changes in a new branch from master. I delete old one, create a complete new branch from master and made the changes. I shouldn't have any conflicts. I don't see any conflicts.

File: ads/bidtellect.js (hunk 1 above)
Comment (edaquinta-bidtellect) [inline ads/bidtellect.js]: ??? Didn't get this comment...

File: ads/_config.js (hunk 2 above)
Comment (edaquinta-bidtellect) [inline ads/_config.js]: I will look into this.

File: ads/bidtellect.js (hunk 1 above)
Comment (zhouyx) [inline ads/bidtellect.js]: `const requiredParams = ['t', 'pid', 'sid'];`

**Comment by zhouyx** [comment]
I guess your master has conflict by the time you create new branch from it.

**Comment by edaquinta-bidtellect** [comment]
This start becoming annoying for me. I can't see conflicts or any new commit. I use SourceTree on Mac, merge master into my brach and push. No conflicts. Pull latest from master, no new commits.

File: ads/bidtellect.js (hunk 1 above)
Comment (edaquinta-bidtellect) [inline ads/bidtellect.js]: Done!

**Comment by zhouyx** [comment]
@edaquinta-bidtellect Sorry I've never used SourceTree, so can't help much.  Please refer to our [getting started doc](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-quick.md#pull-the-latest-changes) for a general guidline.

**Comment by edaquinta-bidtellect** [comment]
git rebase master
Current branch bidtellect is up to date.

**Comment by jridgewell** [comment]
You'll need to rebase off of the upstream master (given you named the `ampproject/amphtml` repo `upstream`):

```bash
git remote add upstream https://github.com/ampproject/amphtml.git
git fetch upstream
git checkout bidtellect
git pull --rebase upstream master
git push -f origin bidtellect
```

**Comment by edaquinta-bidtellect** [comment]
Let me know now... Thanks @jridgewell

**Comment by jridgewell** [comment]
Merge conflicts are gone, now just waiting for Travis and the CLA Bot to OK.

**Comment by edaquinta-bidtellect** [comment]
Travis failed. Not sure If I need to do something.

**Comment by edaquinta-bidtellect** [comment]
https://cdn.ampproject.org is failing

**Comment by jridgewell** [comment]
The failure is unrelated. We'll fix this on master, but you'll need to rebase again after it's merged. Sorry.

/cc @rsimha-amp

**Comment by edaquinta-bidtellect** [comment]
God bless me... lol

**Comment by jridgewell** [comment]
Worse case, we can force merge even with this failure.

**Comment by rsimha-amp** [comment]
Sorry about that. I've whitelisted the false negative failure in #9596

**Comment by jridgewell** [comment]
Awesome, do you mind rebasing one more time? Sorry about all the trouble.

**Comment by edaquinta-bidtellect** [comment]
Done! Still no CLA though.

**Comment by jridgewell** [comment]
Have you confirmed the CLA on your side? Maybe you're missing some info? If not, I'll escalate with the CLA reviewers.

**Comment by edaquinta-bidtellect** [comment]
I have no idea about CLA. I only know we signed a month ago and I'm sure I'm in the google group as an owner. But other than that I don't know nothing else. Waiting all this time without any feedback.

**Comment by jridgewell** [comment]
Alright, I've escalated. They're generally pretty responsive (a few hours at most), so we should be able to track down the issue shortly.

**Comment by edaquinta-bidtellect** [comment]
Thanks!

**Comment by jridgewell** [comment]
Seems Kim Bamback needs to sign the CLA. An email was sent to her with a online form. They just resent.

**Comment by edaquinta-bidtellect** [comment]
She signed already. Today in the morning (ET). Any updates?

**Comment by jridgewell** [comment]
None on my end, should't be long though.

**Comment by edaquinta-bidtellect** [comment]
CLA is done! Anything else?

**Comment by jridgewell** [comment]
Yay! @zhouyx needs to approve.

**Comment by edaquinta-bidtellect** [comment]
You can merge it! I did not implemented renderStart.

**Comment by zhouyx** [comment]
Thanks for the Pull Request. Merged.

**Comment by edaquinta-bidtellect** [comment]
Thanks @jridgewell and @zhouyx. Sorry for the merge issues ;)

**Pull request merged**